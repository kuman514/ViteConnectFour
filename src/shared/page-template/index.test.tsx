import { render } from '@testing-library/react';
import { createRoutesStub } from 'react-router';

import PageTemplate from '.';

describe('PageTemplate', () => {
  it('should match with snapshot', () => {
    const Stub = createRoutesStub([
      {
        path: 'pages',
        Component: PageTemplate,
        children: [
          {
            path: 'koishi',
            Component: () => <div>Aggie Koishi</div>,
          },
        ],
      },
    ]);
    const { container } = render(<Stub initialEntries={['/pages/koishi']} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
