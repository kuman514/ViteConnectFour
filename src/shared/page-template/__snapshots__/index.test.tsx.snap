// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`PageTemplate should match with snapshot 1`] = `
<main
  aria-label="page"
  class="w-screen h-dvh flex justify-center items-center"
>
  <section
    aria-label="page-body"
    class="w-full h-full max-w-[100dvh] max-h-[100vw] aspect-square"
  >
    <div>
      Aggie Koishi
    </div>
  </section>
</main>
`;
